.github/ISSUE_TEMPLATE/GameBalanceForm.yml

```
@@ -0,0 +1,33 @@
+name: Game balance
+description: Sugguest changes to the ballance of the mission.

```

You found an extra L here
```suggestion
description: Sugguest changes to the balance of the mission.

```

---

balance is actually correct, but there is an "u" to many in Suggest 😄

---

:P